pkg/windows/build_env_3.ps1

```diff
@@ -276,6 +276,15 @@ ForEach($key in $ini[$bitDLLs].Keys) {
     Copy-Item $file  -destination $($ini['Settings']['Python3Dir'])
 }
 
+#------------------------------------------------------------------------------
+# Remove the temporary download directory
+#------------------------------------------------------------------------------
+Write-Output " ----------------------------------------------------------------"
+Write-Output " - $script_name :: Cleaning up downloaded files"
+Write-Output " ----------------------------------------------------------------"
+Write-Output ""
+Remove-Item $($ini['Settings']['DownloadDir']) -Force -Recurse
+
```

Comment:
Can you make the same change in the `build_env_2.ps1` file so that the are basically the same?

Reply:
I switched it back to normal. Someone might want it to be in that order.